docs/theming.md

```diff
@@ -185,9 +185,174 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+This migration guide covers the typical migration path from Themer usage to Theming extensions.
+
+Theming extensions are discussed in detail above. Themers are discussed in the [Themers](#themers)
```

Comment:
Please make this hyperlink be a bit more descriptive than the single-word "Themers".  Consider something like:

> Theming extensions are discussed in detail above. For [more information about Themers](#themers) see the section below.